### ISSUE TITLE
Map images breaking on server restart

### ISSUE THREAD

**Issue by BlitzGamer-88** [RockinChaos/ItemJoin#311]
So I've just started using this plugin and I found the feature of adding images on maps very interesting so I decided to use it. I added my server logo to the folder made an item which can be seen bellow and took the logo and made resized it to 128 x 128 pixels. The map works until I restart the server. When I do `/stop` or I press the stop button from the console start the server again the map resets and I get this: https://i.imgur.com/qOiF2Ye.png which is a starting of a rendering of my lobby/spawn. To reset this I need to clear my inventory. It does happen when I am and when I am not op.

I've made this item: 

```
  server-logo:
    id: FILLED_MAP
    custom-map-image: server-icon.png
    slot: 4
    name: '&ePlay...'
    itemflags: item-store, item-craftable, item-modifiable, unbreakable, hide-attributes, death-drops, self-drops, placement, inventory-modify
    triggers: join, respawn, world-switch, region-enter
    enabled-worlds: world
```
then restarted the server

Items.yml: https://paste.helpch.at/nemagifitu.php
Config.yml: https://paste.helpch.at/ozilanazoy.php
Photo proprieties: https://imgur.com/a/aJYnrlv

Server version:
```
[01:28:29 INFO]: This server is running Paper version git-Paper-83 (MC: 1.16.1) (Implementing API version 1.16.1-R0.1-SNAPSHOT)
[01:28:29 INFO]: You are 3 version(s) behind
```
ItemJoin version:
```
[01:29:01 INFO]: ItemJoin v5.0.6-RELEASE-b395 by RockinChaos
[01:29:01 INFO]: Type /ItemJoin Help for the help menu.
```

**Comment by RockinChaos** [comment]
First off THANK YOU for the fantastic bug writeup!

Map items are known to have major issues with map items in v5.0.6 and have had A LOT of changes in the latest snapshot. I would recommend trying that and seeing if the issue resolves itself.

If the issue continues please try to reset your map images via `/itemjoin purge map-ids server-icon.png` this command was implemented in the latest snapshot to reset ID's assigned to map items in the case another plugin started using that id.

Do note: in both cases, you will need to re-get the map item since it is a bugged item from v5.0.6. (Although after regretting the item, upon restart/rejoin the new map item given should function without breaking).

Snapshot; http://ci.craftationgaming.com/job/ItemJoin/472/

Let me know!

**Comment by BlitzGamer-88** [comment]
Hello so I've updated to the dev builds:

ItemJoin v5.0.7-SNAPSHOT-b472.

I've removed the item from my inventory and rejoin (also restarted the server of course). The map works until I restart the server again. I've also purged the map id but the same thing happens. I've also purged the default.jpg photo as well just in case but doesn't seem to help.

**Comment by RockinChaos** [comment]
Okay so, after some extensive testing I was able to replicate and fix the issue on my end. It seems to have been an issue with database syncing at the initial startup.

Here is the fixed snapshot---let me know: http://ci.craftationgaming.com/job/ItemJoin/480/

It's very likely you will need to purge the map images again and /clear to re-get the item. Thanks for your patience in debugging this issue.

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.0.7.
You can grab it here; Download: https://www.spigotmc.org/resources/itemjoin.12661/download?version=354306

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
